<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>801</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="text">
        <string>Select Excel Files</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>721</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>681</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Copy From:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edit_src">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_src_browse">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Destination:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edit_dst">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_dst_browse">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>580</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_run">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>520</y>
      <width>300</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>290</y>
      <width>201</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="btn_new">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_edit">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_remove">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>261</y>
      <width>491</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Macro</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="list_macro"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_src_browse</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>browser_src()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>713</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>711</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_dst_browse</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>browser_dst()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>735</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_new</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>create_macro()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>664</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>760</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_edit</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>edit_macro()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>783</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_remove</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>remove_macro()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>793</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_run</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>run_macro()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>797</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_macro</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>select_macro(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>858</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browser_src()</slot>
  <slot>browser_dst()</slot>
  <slot>create_macro()</slot>
  <slot>edit_macro()</slot>
  <slot>remove_macro()</slot>
  <slot>run_macro()</slot>
  <slot>select_macro(QListWidgetItem*)</slot>
 </slots>
</ui>
